lib_xua/lib_build_info.cmake

```diff
@@ -20,14 +20,14 @@ set(LIB_INCLUDES api
                  src/hid
                  src/midi)
 set(LIB_OPTIONAL_HEADERS xua_conf.h static_hid_report.h)
-set(LIB_DEPENDENT_MODULES "lib_locks"
-                          "lib_logging"
-                          "lib_mic_array(feature/xcommon_cmake)"
+set(LIB_DEPENDENT_MODULES "lib_adat(1.1.0)"
+                          "lib_locks(2.2.0)"
+                          "lib_logging(3.2.0)"
+                          "lib_mic_array(4.6.0)"
                           "lib_spdif"
-                          "lib_xassert"
-                          "lib_xud"
-                          "lib_adat"
-                          "lib_sw_pll")
+                          "lib_sw_pll(2.1.0)"
+                          "lib_xassert(4.2.0)"
+                          "lib_xud")
```

Comment:
2.2.3 in module_build_info. Any reason for the diff?

Reply:
lib_xud hasn't been released yet with xcommon_cmake support for this upcoming release. Then both module_build_info and lib_build_info.cmake will be 2.3.0.